ci-operator/step-registry/cucushift/installer/check/nutanix/zones/cucushift-installer-check-nutanix-zones-commands.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+
+declare prism_element1
+declare prism_element2
+declare prism_element3
+
+# shellcheck source=/dev/null
+source "${NUTANIX_AUTH_PATH}"
+
+node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
+
+if [ "$SINGLE_ZONE" ]; then
```

Comment:
Can we also print out some info about the checking is running against single zone or multi zones cluster in the if and else branch?